docs/src/main/asciidoc/command-mode-reference.adoc

```diff
@@ -112,20 +112,21 @@ thread (A non-command mode application is essentially just running an applicatio
 If you want to shut down a running application and you are not in the main thread then you should call `Quarkus.asyncExit`
 in order to unblock the main thread and initiate the shutdown process.
 
-=== Dev mode
+=== Development Mode
 
-Also for command mode applications the dev mode is supported. When running `mvn compile quarkus:dev`, the command mode application is executed and on press of the `Space bar` key, is restarted.
+Also for command mode applications the dev mode is supported. When you run `mvn quarkus:dev`, the command mode application is executed.
 
-As command mode applications will often require arguments to be passed on the commandline, this is also possible in dev mode via:
+As command mode applications will often require arguments to be passed on the commandline, this is also possible in dev mode via the `quarkus.args` system property. For example `mvn quarkus:dev -Dquarkus.args='--help'` and the same can be achieved with Gradle: `./gradlew quarkusDev --quarkus-args='--help'`.
```

Comment:
Should add the CLI equivalent, while you're at it: `quarkus dev -- args to pass along`
(they would then be updated in the same way)